.github/workflows/pr-check-build-bundlesize.yml

```diff
@@ -0,0 +1,19 @@
+name: PR Check Build
+on: pull_request
+
+jobs:
+  pr-check-bundlesize:
```

Comment:
🔶 suspect this is a copy-paste issue. This doesn't have anything to do with bundlesize.